packages/core/src/exec.ts

```diff
@@ -87,12 +87,8 @@ function getActionFunction<TState extends AnyState>(
         interpreter.defer(sendAction);
         return;
       } else {
-        if (sendAction.params.to) {
-          const target = sendAction.params.to;
-          execSendTo(sendAction.params._event, target, actorCtx);
-        } else {
-          interpreter.send(sendAction.params._event as SCXML.Event<any>);
```

Comment:
The notable difference is now that events sent to self will also get `.origin`. I think this wasn't the case before this change